Pawfect-Harmony/src/main/java/org/launchcode/Pawfect/Harmony/PawfectHarmonyApplication.java

```diff
@@ -4,11 +4,13 @@
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 
-@SpringBootApplication (exclude = {DataSourceAutoConfiguration.class })
```

Comment:
Same thing. Why was this changed? I am just curious.

Reply:
I had to take out the (exclude = {DataSourceAutoConfiguration.class }) in order to get the program to try to connect to a database. I had originally put that line in because I was trying to develop pages before I connected things to mySQL but I kept getting an error about not connecting to a database and I discovered that if you add that exclude statement after @SpringBootApplication it will keep it from trying to connect to a database. It was useful when I had no database, but I had to remove it once I got a database up and running to use.

Reply:
You mentioned this in your slack messages, and talked about this several times in class. I should have reviewed them before looking at this pull request. I will do that in the future.